lib/archethic/mining/fee.ex

```diff
@@ -42,11 +42,8 @@ defmodule Archethic.Mining.Fee do
       true == Transaction.network_type?(type) ->
         0
 
-      type == :oracle ->
-        0.01 / uco_price_in_usd
-
       true ->
-        transaction_value = get_transaction_value(tx) / @unit_uco
+        transaction_value = 0.01 / uco_price_in_usd
```

Comment:
I think you should make the change here:

https://github.com/archethic-foundation/archethic-node/blob/3c24793b3c00085b5f3ccfabafe8ca557a604e99/lib/archethic/mining/fee.ex#L99

As there is no more transaction value fee